Gruntfile.js

```diff
@@ -178,6 +178,7 @@ module.exports = function(grunt) {
           'i18n/**/*.js',
           '!docs/app/assets/js/angular-bootstrap/**',
           '!docs/config/templates/**',
+          '!docs/bower_components/**',
```

Comment:
Is that still needed?

Reply:
hmm, well it was causing a problem locally but then perhaps I had some old unused files ...

Reply:
removing